src/Novu/DTO/EventTriggerDataDto.cs

```diff
@@ -17,13 +17,17 @@ public class EventTriggerDataDto
     [JsonProperty("payload")]
     public object Payload { get; set; } = new();
 
+
     [JsonProperty("override")] public List<object>? Overrides { get; set; }
     
     /// <summary>
     /// A unique identifier for this transaction, will generated a UUID if not provided.
     /// </summary>
     [JsonProperty("transactionId")]
     public string? TransactionId { get; set; }
+
+    [JsonProperty("actor")]
+    public string Actor { get; set; }
```

Comment:
This property should be able to accept either a string or an object. @wh1337 what's the best way to represent it?

Reply:
That's fine for me. We just need to update the sdk documentation to let people know.